doc/user/getting-help.rst

```diff
@@ -0,0 +1,73 @@
+.. vim: set filetype=rst
+
+===============
+How to get help
+===============
+
+First, be sure that you:
+
+#. Read the documentation (this site)
+
+#. Work through relevant examples in the `khmer-recipes repository
+   <https://github.com/ged-lab/khmer-recipes>`__. Find **khmer-recipes**
+   `documentation here <http://khmer-recipes.readthedocs.org/en/latest/#>`__.
+
+#. Google search for the error output and/or keywords related to your problem.
+   Here you can search results from the mailing list, where others may
+   have discussed solutions to the same issue.
+
+.. raw:: html
+
+    <form action="http://google.com/search" method="get">
+    <input type="text" name="q" size="28" maxlength="255" value="" />
+    <input type="submit" value="Google Search" />
+    <br/>
+    <input type="checkbox" name="sitesearch"
+    value="http://lists.idyll.org/pipermail/khmer" checked /> only search
+    khmer discussion email archive<br/>
+    </form>
+
+Mailing List
+------------
+
+The primary way to get help is through the khmer discussion list:
+http://lists.idyll.org/listinfo/khmer
+
+When you ask a question, it is important that you ask in a way that ensures
+you get the best answer possible and minimizes back-and-forth with the good
+people trying to help.
+
+Asking a good question
+----------------------
+
+#. Be very specific in your subject line.
```

Comment:
Given the low volume of requests we receive lets not add extra requirements for asking a question at this time.